app_dart/lib/src/service/branch_service.dart

```diff
@@ -30,6 +34,44 @@ class BranchService {
   final GerritService gerritService;
   final RetryOptions retryOptions;
 
+  /// Add a [CreateEvent] branch to Datastore.
+  Future<void> handleCreateRequest(CreateEvent createEvent) async {
+    log.info('the branch parsed from string request is ${createEvent.ref}');
+
+    final String? refType = createEvent.refType;
+    if (refType == 'tag') {
+      log.info('create branch event was rejected because it is a tag');
+      return;
+    }
+    final String? branch = createEvent.ref;
+    if (branch == null) {
+      log.warning('Branch is null, exiting early');
```

Comment:
umm just to clarify, does debug level log mean `log.severe`? (based on the example in https://pub.dev/packages/logging)

For context, this file we just restored it from the PR that deleted it, without changing it.

Reply:
Changed to log.fine.

Umm for my learning purposes, why do we want to use `log.fine` instead of `log.warning`. My understanding is that if branch is null, something went wrong and we would like to raise a warning. based on doc, it seems like log.warning is used as key for potential problems, and log.fine is used as key for tracing information.

Reply:
synced with Casey. When the request sent is not the expected format, log the info but do not crash, since 99 percent of the time it is in the correct format